saleor/tests/e2e/transactions/utils.py

```diff
@@ -0,0 +1,64 @@
+from ..utils import get_graphql_content
+
+TRANSACTION_CREATE_MUTATION = """
+mutation TransactionCreate($id: ID!, $transactionCreateInput: TransactionCreateInput!) {
+  transactionCreate(id: $id, transaction: $transactionCreateInput) {
+    errors {
+      field
+      message
+      code
+    }
+    transaction {
+      id
+      name
+      message
+      pspReference
+      actions
+      chargedAmount {
+        currency
+        amount
+      }
+    }
+  }
+}
+"""
+
+
+def create_transaction(
+    e2e_api_client,
+    id,
+    transaction_name="Credit card",
+    message="Charged",
+    psp_reference="PSP-ref123",
+    available_actions=["CANCEL", "REFUND"],
+    currency="USD",
+    amount=1,
```

Comment:
I would suggest to create a separate functions for fully paid and partially paid order. Alternatively I would add the variable `amount` in your test as currently you use create_transaction(api_client, id) without providing the amount for a partial payment. IMO it's confusing and not clear as a person who reads the test code have to go to utils to ensure the insufficient value was used for the payment.